OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -55,6 +55,10 @@ public abstract class AttackBaseInfo : PausableConditionalTraitInfo
 		[Desc("Tolerance for attack angle. Range [0, 512], 512 covers 360 degrees.")]
 		public readonly WAngle FacingTolerance = new(512);
 
+		[Desc("Show the target cursor even on empty terrain cells when enabled.",
+			"Any traits implementing IMove must be absent or disabled for this to work.")]
```

Comment:
But it works just fine? You can still order around units with force move in necessary

Reply:
Depends on how you setup the conditions, but yes. See new testcase commit.

Reply:
it's still wrong to keep this comment

Reply:
Oh, I see now. You're original comment wasn't a question but a statement. I don't like resorting to force-move, so this is really meant to be used with conditions, hence the comment. Removed.